pootle/apps/pootle_store/models.py

```diff
@@ -1294,14 +1288,10 @@ def toggle_qualitycheck(self, check_id, false_positive, user):
             sub_type = SubmissionTypes.UNMUTE_CHECK
 
         sub = Submission(creation_time=timezone.now(),
-            translation_project=self.store.translation_project,
-            submitter=user,
-            field=SubmissionFields.NONE,
-            unit=self,
-            store=self.store,
-            type=sub_type,
-            quality_check=check
-        )
+                         translation_project=self.store.translation_project,
+                         submitter=user, field=SubmissionFields.NONE,
+                         unit=self, store=self.store, type=sub_type,
+                         quality_check=check)
```

Comment:
@unho either are fine and valid.  I wans't primarily trying to increase readability around the long list of parameters dimension.  Primarily my aim is PEP8 which generally then does increase readability, something I'm actually finding with long lines that nobody wanted to fold.  Once I fold them I find I can read the statement without tracking my eyes.

The original has its own problems in that creation_time should also be wrapped in which case it's a proper list.  I'd rather leave the ideas of is this OK in the fix vs should it be a list to the next person who needs to work on the code.  As in some cases vertical space is harder to read then horizontal space.